.grabl/automation.yml

```diff
@@ -28,19 +28,17 @@ build:
       command: |
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
-        bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=streamed
+        bazel test $(bazel query 'kind(checkstyle_test, //...)')
     test-unit:
       image: vaticle-ubuntu-20.04
       command: |
         bazel test $(bazel query 'kind(kt_jvm_test, //...) except kind(kt_jvm_test, //test/...)')
     test-integration:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel run //:typedb-extractor -- dist/typedb-all-linux
-        sudo systemd-run ./dist/typedb-all-linux/typedb server
-        sleep 60
         bazel build //test/integration/...
-        bazel test $(bazel query 'kind(kt_jvm_test, //test/integration/...)') --test_output=streamed --spawn_strategy=local
+        bazel test //test/integration/... --jobs=1 --spawn_strategy=local
```

Comment:
I realised we had a number of `typedb_kt_test`s now instead of just `kt_jvm_test`. I think bazel was smart enough to tell this, but I'm making this more explicit so that we don't accidentally miss this in the future.

Reply:
I'm still keen to keep the logic of this 'run every test except `checkstyle_test`s.

So, we ended up with 
```
bazel test $(bazel query '//test/integration/... except kind(checkstyle_test, //test/integration/...)')
```